http/query_handler.go

```diff
@@ -104,6 +109,11 @@ func (h *FluxHandler) handleQuery(w http.ResponseWriter, r *http.Request) {
 	defer span.Finish()
 
 	ctx := r.Context()
+	if id, sampled, found := logger.TraceInfo(ctx); found {
+		println(id, sampled)
```

Comment:
remove the debug print

Reply:
whoops, thanks! there are two instances of that, ill remove them